docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/ComponentControlsCodeSandbox.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react'
+import CodeSandboxer from 'react-codesandboxer'
+import { Menu } from 'semantic-ui-react'
+
+import pkg from 'package.json'
+import { ComponentSourceManagerLanguage } from 'docs/src/components/ComponentDoc/ComponentSourceManager'
+import { updateForKeys } from 'docs/src/hoc'
+import { appTemplateJs } from './indexTemplates'
+
+const dependencies = {
+  '@stardust-ui/react': pkg.version,
```

Comment:
would suggest to consume dependency names from the same source that is used by `importsResolver` of `SourceRender` - to ensure that they always will be in sync 👍

Reply:
It's impossible because these deps will fetched from NPM on CodeSandbox side 😭

Reply:
yes, but speaking of the keys - those could be kept in sync, right? (like `react`, `lodash`, etc)

Reply:
Oh, I understand your point, will do